### ISSUE TITLE
New label,  HP:0005942: intra-alveolar accumulation of macrophages

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6440]
**HPO term**
Desquamative interstitial pneumonitis HP:0005942

**Preferred term label:**
intra-alveolar accumulation of macrophages
(maybe better to differentiate between histological term and disease term?)

**Synonym to be added**
Filling of the alveoli with alveolar macrophages, Desquamative interstitial pneumonitis
Filling of the alveoli with “desquamated” epithelial cells

**Parent term (use hpo.jax.org/app)**
Abnormal pulmonary alveolar system morphology HP:0033137

**broad/exact/narrow/plural/abbreviation ?**

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
leave DIP as synonym

**Comment by mgriese** [comment]
ok. 
Would introduce that interstitial pneumonitis is synonym to interstitial pneumonia

**Comment by pnrobinson** [comment]
preferred label of DIP
put DIP as related synonym

**Comment by pnrobinson** [comment]
synonym --  intra-alveolar accumulation of macrophages

**Comment by pnrobinson** [comment]
added synonyms
